tests/voq/fabric_data/Arista-7800R3-48CQ2-C48_Arista-7804R3-FM_LC3.yaml

```diff
@@ -0,0 +1,450 @@
+---
+asic0:
+  0:
+    peer asic: '1'
+    peer lk: '133'
+    peer slot: 3
```

Comment:
To be consistent, shouldn't peer_slot field also be a string, instead of an int.